scripts/process_spraggins.sh

```diff
@@ -8,64 +8,70 @@ main() {
+    SERVER_URL="https://vitessce-data.storage.googleapis.com/$RELEASE/spraggins/"
 
     echo "Download and process IMS data..."
 
-    SPRAGGINS_URL="https://vitessce-data.s3.amazonaws.com/source-data/spraggins"
-
-    [ -e "$IMZML_IN" ] || \
-        wget "$SPRAGGINS_URL/spraggins.ims.imzml" -O "$IMZML_IN"
-    [ -e "$IBD_IN" ] || \
-        wget "$SPRAGGINS_URL/spraggins.ims.ibd" -O "$IBD_IN"
+    [ -e "$IMS_IMZML_IN" ] || \
+        wget "$SOURCE_URL/spraggins.ims.imzml" -O "$IMS_IMZML_IN"
+    [ -e "$IMS_IBD_IN" ] || \
+        wget "$SOURCE_URL/spraggins.ims.ibd" -O "$IMS_IBD_IN"
 
-    if [ -e "$ZARR_OUT" ]
+    if [ -e "$IMS_ZARR_OUT" ]
     then
-        echo "Skipping zarr -- output already exists: $ZARR_OUT"
+        echo "Skipping zarr -- output already exists: $IMS_ZARR_OUT"
     else
         echo 'Generating IMS zarr may take a while...'
         CMD="$BASE/python/imzml_reader.py
-            --imzml_file $IMZML_IN
-            --ibd_file $IBD_IN
-            --ims_zarr $ZARR_OUT
-            --ims_metadata $JSON_OUT
-            --zarr_store_url $ZARR_STORE_URL"
+            --imzml_file $IMS_IMZML_IN
+            --ibd_file $IMS_IBD_IN
+            --ims_zarr $IMS_ZARR_OUT
+            --raster_json $IMS_JSON_OUT
+            --raster_name '$IMS_RASTER_NAME'
+            --server_url $SERVER_URL"
         echo "Running: $CMD"
         eval $CMD
     fi
 
-    TILES_BASE='spraggins.images'
-    TILES_PATH="$OUTPUT/$TILES_BASE"
-    if [ -e "$TILES_PATH" ]
+    if [ -e "$MXIF_ZARR_OUT" ]
     then
-        echo "Skipping tiling -- output already exists: $TILES_PATH"
+        echo "Skipping tiling -- output already exists: $MXIF_ZARR_OUT"
     else
-        if [ -e "$INPUT/spraggins.ome.tif" ]
+        if [ -e "$MXIF_TIFF_IN" ]
         then
-          echo "Not copying $INPUT/spraggins.ome.tif from s3 - already exists or testing"
+          echo "Not copying $MXIF_TIFF_IN from s3 - already exists or testing"
         else
-          aws s3 cp s3://vitessce-data/source-data/spraggins/spraggins.ome.tif "$INPUT/spraggins.ome.tif"
+          wget "$SOURCE_URL/spraggins.ome.tif" -O "$MXIF_TIFF_IN"
         fi
-        RELEASE=${CLOUD_TARGET//vitessce-data\//}
-        SERVER_URL="https://vitessce-data.storage.googleapis.com/$RELEASE/spraggins/"
-        CMD='docker run --rm
-            -e "SERVER_URL='$SERVER_URL'"
-            -e "PREFIX=spraggins"
-            -e "PYRAMID_TYPE=tiff"
-            --mount "type=bind,src='$INPUT'/spraggins.ome.tif,destination=/input.ome.tif"
-            --mount "type=bind,src='$OUTPUT',destination=/output_dir"
-            --name tiler gehlenborglab/ome-tiff-tiler:v0.0.7'
+        echo 'Converting OME-TIFF to zarr may take a while...'
+        CMD="$BASE/python/ome_tiff_reader.py
+            --input_tiff $MXIF_TIFF_IN
+            --output_zarr $MXIF_ZARR_OUT
+            --raster_json $MXIF_JSON_OUT
+            --raster_name '$MXIF_RASTER_NAME'
```

Comment:
Quotes are always safer, but is there a reason to be concerned about this one, and not the others?